.github/workflows/ci.yml

```diff
@@ -18,19 +18,11 @@ jobs:
       fail-fast: false
       matrix:
         ruby:
-          - '2.7'
-          - '3.0'
           - '3.1'
           - '3.2'
           - '3.3'
         gemfile:
-          - gemfiles/rails-7.1.0.gemfile
-          - gemfiles/rails-7.2.0.beta2.gemfile
-        exclude:
-          - ruby: '2.7'
-            gemfile: gemfiles/rails-7.2.0.beta2.gemfile
-          - ruby: '3.0'
-            gemfile: gemfiles/rails-7.2.0.beta2.gemfile
```

Comment:
I'd argue that we can't drop as many ruby versions because we're beholden to what rails supports.